pkgs/tools/misc/ethminer/default.nix

```diff
@@ -45,6 +45,7 @@ in stdenv.mkDerivation rec {
     "-DCMAKE_BUILD_TYPE=Release"
   ] ++ lib.optionals (!cudaSupport) [
     "-DETHASHCUDA=OFF" # on by default
+    "-DCUDA_PROPAGATE_HOST_FLAGS=off"
```

Comment:
This line needs to go after line 45 so it is passed when cuda is enabled, not when it isn't. Also, remove line 85, the `broken = cudaSupport` one.

Reply:
Would probably be best to just put all of that in a ternary; i.e.

`++ if cudaSupport then [ ... ] else [ ... ];

Reply:
Overlooked, sorry :)